app/controllers/admin/delivery_partners/users_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Admin
+  module DeliveryPartners
+    class UsersController < Admin::BaseController
```

Comment:
I don't think we want to inherit from `Admin::BaseController` or be in the `Admin` module. This area is not an "admin" area. its just an area for DP users. so could we split this out?